Client/Telemetry/UnifiedTelemetry.swift

```diff
@@ -115,6 +115,10 @@ class UnifiedTelemetry {
 
             outputDict["settings"] = settings
 
+            let delegate = UIApplication.shared.delegate as? AppDelegate
+   
+            outputDict["openTabCount"] = delegate?.tabManager.count
```

Comment:
Concerned this will put in an `Int?` into the dict, maybe `delegate?.tabManager.count ?? 0` to ensure we get an unwrapped value. I think we do dumb stringification of values using `.description` so optionals will print as `Optional<Int>(value)`